lib/galaxy/webapps/reports/controllers/users.py

```diff
@@ -178,7 +178,9 @@ def user_disk_usage(self, trans, **kwd):
 
         user_cutoff = int(kwd.get('user_cutoff', 60))
         # disk_usage isn't indexed
-        users = sorted(trans.sa_session.query(galaxy.model.User).all(), key=operator.attrgetter(str(sort_id)), reverse=_order)
+        all_users = trans.sa_session.query(galaxy.model.User).all()
+        sortAttrGetter = operator.attrgetter(str(sort_id))
```

Comment:
```suggestion
        sort_attrgetter = operator.attrgetter(str(sort_id))
```

Reply:
I forgot to take my javascript hat back off for a minute there, sure :)